app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -0,0 +1,63 @@
+// This file loads Bootstrap by importing components explicitly instead of doing `@import 'bootstrap';`
```

Comment:
This file is not redux specific, but it's included to show you how to incorporate twitter bootstrap in your project so that it works with the Webpack Dev Server (node server) on port 4000 for quick testing.